doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/architecture-scenarios-ref.adoc

```diff
@@ -0,0 +1,89 @@
+[id='architecture-scenarios-ref_{context}']
+= Example decision management architectures with {PRODUCT}
```

Comment:
an architecture reference without a diagram is not a good architecture reference to me.

Reply:
I actually agree with you and wondered the same. So do you think a diagram of each of these 4 examples would be good? Something showing a flow from installation to authoring to deployment and execution? Do you have a quick sketch you recommend as an example? You can just sketch out what you thing on a paper or a white board and take a picture to share and I can work with that. Otherwise I'll shoot in the dark and see what you think :)

Reply:
I'm sorry, but you're on your own on this.. i can certainly review later.

Reply:
Sure. I'll come up with something and send it your way at some point, but it'll be in a separate effort outside of this PR. I don't think we need to hold publishing this for that reason. That okay with you?